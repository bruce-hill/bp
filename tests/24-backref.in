foo baz foo
xx yy xx
x x x
foo baz xx
a b c
