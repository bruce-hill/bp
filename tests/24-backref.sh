# With @-capturing, you can reference previous captures
# Example: bp -p '@first=`a-z .. first' matches "aba" and "xyzx", but not "abc"
bp -p '@first=+Abc _ +Abc _ first'
