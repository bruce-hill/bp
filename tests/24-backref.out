foo baz foo
xx yy xx
x x x
